Makefile

```diff
@@ -409,7 +409,7 @@ start-watcher_info:
 	echo "Potential cleanup" && \
 	rm -f ./_build/${BAREBUILD_ENV}/rel/watcher_info/var/sys.config || true && \
 	echo "Init Watcher Info DBs" && \
-	_build/${BAREBUILD_ENV}/rel/watcher_info/bin/watcher_info eval "OMG.DB.ReleaseTasks.InitKeyValueDB.run()" && \
+	_build/${BAREBUILD_ENV}/rel/watcher_info/bin/watcher_info eval "OMG.Watcher.ReleaseTasks.InitKeyValueDB.run_multi()" && \
```

Comment:
I keep it under Watcher to make it clear that Watcher-Info requires an separate instance of Watcher running in it own erlang vm (they communicate in the same VM via OMG.Bus).